jetty-core/jetty-http3/jetty-http3-server/src/main/java/org/eclipse/jetty/http3/server/internal/HTTP3StreamServer.java

```diff
@@ -50,22 +55,20 @@ public void onRequest(HeadersFrame frame)
         validateAndUpdate(EnumSet.of(FrameState.INITIAL), FrameState.HEADER);
         onHeaders(frame);
         updateClose(frame.isLast(), false);
-        listener = notifyRequest(frame);
+        listener = getSession().notifyRequest(frame);
```

Comment:
Can `notifyRequest()` return null? Should it? I find the usage of null unclear here.